cmd/iam.go

```diff
@@ -205,6 +205,28 @@ func (sys *IAMSys) SetTempUser(accessKey string, cred auth.Credentials) error {
 	return nil
 }
 
+// ListUsers - list all users.
+func (sys *IAMSys) ListUsers() (map[string]madmin.UserInfo, error) {
+	objectAPI := newObjectLayerFn()
+	if objectAPI == nil {
+		return nil, errServerNotInitialized
+	}
+
+	var users = make(map[string]madmin.UserInfo)
+
+	sys.RLock()
+	defer sys.RUnlock()
+
+	for k, v := range sys.iamUsersMap {
+		users[k] = madmin.UserInfo{
+			SecretKey: v.SecretKey,
```

Comment:
I guess it is not really needed to send our users secret keys here, at least right now, right ?

Reply:
It is sent on purpose @vadmeste to know what was set, otherwise I don't see a value of this operation.

Reply:
> It is sent on purpose @vadmeste to know what was set, otherwise I don't see a value of this operation.

I mean we probably can:
- set a new password
- enabled/disabled user

But it won't be necessary to show users passwords

Wouldn't that work?

Reply:
Fixed it